### PR TITLE
fix(node): tag some TODOs to created issues

### PR DESCRIPTION
**Pull request by lrubiorod**

### CODE REVIEW

File: node/src/actors/chain_manager/mining.rs [hunk 1]
```diff
@@ -72,18 +72,14 @@ impl ChainManager {
             _ => return,
         };
 
-        if beacon.checkpoint > current_epoch {
+        if beacon.checkpoint >= current_epoch {
```
Comment (aesedepece) [inline node/src/actors/chain_manager/mining.rs]: Undocumented edit. What are you trying to conceal? :smirk:

Comment (tmpolaczyk) [reply node/src/actors/chain_manager/mining.rs]: He's trying to introduce a backdoor :scream:

Comment (lrubiorod) [reply node/src/actors/chain_manager/mining.rs]: Chain checkpoint beacon is the checkpoint beacon of the last consolidated block. Due to consolidate a block from epoch N is done in epoch N+1, our chain beacon always come from the past epoch. A chain beacon with the current epoch is the same error if it is come from the future

Comment (tmpolaczyk) [reply node/src/actors/chain_manager/mining.rs]: Then put this change in a separate commit :P

File: node/src/actors/chain_manager/mining.rs [hunk 2]
```diff
@@ -72,18 +72,14 @@ impl ChainManager {
...
             // We got a block from the future
             error!(
                 "The current highest checkpoint beacon is from the future ({:?} > {:?})",
```
Comment (tmpolaczyk) [inline node/src/actors/chain_manager/mining.rs]:
```suggestion
                "The current highest checkpoint beacon is from the future ({:?} >= {:?})",
```

And add a comment explaining why >= is the same as >

**Pull request merged**